wls-gui-wahllokalsystem/src/components/common/inputs/BaseDateInput.vue

```diff
@@ -0,0 +1,36 @@
+<template>
+  <v-text-field
+    :model-value="modelValue?.toISOString().split('T')[0]"
+    :rules="[DATE_REQUIRED]"
```

Comment:
bitte noch eine rule oder validation einbauen um zu verhindern, dass das datum früher als der wahltag sein kann. alternativ zu einer neuen rule  geht das auch über das prop, das würde ich dann aber so machen dass man von außen das entsprechende datum übergeben kann:

```suggestion
    :rules="[DATE_REQUIRED]"
    min="2026-01-01"
```

Reply:
müssten wir, wenn wir das machen, dann nicht dynamisch einen wahltag nehmen anstatt das date zu hard coden?

Reply:
ja müssen wir. das meinte ich mit 
> dass man von außen das entsprechende datum übergeben kann

das datum war nur als beispiel wie man es umsetzen kann